frameworks/hdfs/src/main/dist/svc.yml

```diff
@@ -213,12 +219,28 @@ pods:
           cp -v zone-resolver.sh {{TASKCFG_ALL_HDFS_VERSION}}/etc/hadoop/zone-resolver.sh
           chmod u+x {{TASKCFG_ALL_HDFS_VERSION}}/etc/hadoop/zone-resolver.sh
           {{/NAME_NODE_PLACEMENT}}
-          ./{{TASKCFG_ALL_HDFS_VERSION}}/bin/hdfs namenode
+          {{#SECURITY_KERBEROS_ENABLED}}
+          sh modify-keytab.sh
+          {{/SECURITY_KERBEROS_ENABLED}}
+          export LATEST_FSIMAGE_VERSION=-65
+          if [[ `cat name-data/current/VERSION | grep "layoutVersion" | cut -d'=' -f2` -gt $LATEST_FSIMAGE_VERSION ]]; then
```

Comment:
can you save the command to a variable that expresses its intent? Also `$()` is the preferred way of wrapping a command over backticks

Reply:
+1

Reply:
Done